common/scripts/make_image.sh

```diff
@@ -39,6 +39,7 @@ TOP_DIR=`pwd`
 PLATDIR=${TOP_DIR}/output
 OUTDIR=${PLATDIR}
 GRUB_FS_CONFIG_FILE=${TOP_DIR}/build-scripts/config/grub.cfg
```

Comment:
Please refactor the changes in file to a new file common/scripts/make_image-arm.sh

The original file common/scripts/make_image.sh shall be unmodified.